### DISCUSSION TITLE
I translated the complete Simplified Chinese version

### DISCUSSION THREAD

**Discussion by huchang47** [Hekili/hekili#1347] [Ideas]
Hekili is really a great plugin!
I translated the complete Simplified Chinese version,and it has been shared to the Chinese WOW community.[This Link](https://nga.178.com/read.php?tid=30198980&_ff=200)
Looking forward to more comprehensive multi-language support,such as local files for easy editing.
I can do all Simplified Chinese localization work,contact me if you need.

**Comment by Hekili** [comment]
Hi, I'd heard about your effort.  When there is more time, I'll re-enable localization so this can be more easily integrated.  Thanks for the link!
